.craft.yml

```diff
@@ -20,3 +21,5 @@ targets:
       pub:sentry_dio:
       pub:sentry_file:
       pub:sentry_sqflite:
+      # This needs to be added to the Sentry registry before uncommenting
+      # pub:sentry_drift:
```

Comment:
We probably also need this for the hive PR, right?

Reply:
Yep, we need to do an initial release first and afterwards add it to the registry so the integration can be retrieved in the docs etc.